lib/fog/compute/google/models/images.rb

```diff
@@ -68,6 +68,25 @@ def get(identity)
           new(data)
         end
 
+        def get_from_family(family)
+          # Search own project before global projects
+          all_projects = [service.project] + global_projects
```

Comment:
While you're in here, could you also update https://github.com/fog/fog-google/blob/b50d2d2a2a815cc664b696c66ea958628172cef7/lib/fog/compute/google/models/images.rb#L11 to look like https://cloud.google.com/compute/docs/images ?